recipes/gappa/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: gappa
+  version: "0.0.0_dev"
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/gavinmdouglas/gappa/archive/v0.0.0-dev.3.tar.gz
+  sha256: c7208e164867827640a52cdad57a200b1b48e81f86224fa875dcedcd23c90602
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - llvm-openmp # [osx]
+  host:
+    - zlib
+  run:
+    - llvm-openmp # [osx]
+
+test:
+  commands:
+    - gappa -h > /dev/null
+
+about:
+  home: https://github.com/lczech/gappa
+  license: GNU General Public License v3.0
+  license_file: LICENSE.txt
+  summary: Genesis Applications for Phylogenetic Placement Analysis
+
+extra:
+  identifiers:
+    - doi:10.1093/bioinformatics/bty767
```

Comment:
If this is the version you used for the publication, you might as well call it `1.0.0`!

Reply:
I'm not a co-author on that paper or a developer of this tool so I think it would be strange for me to set version number.

Reply:
Sorry - I would have sworn I saw your user name in the source url...

Reply:
Any chance to ask upstream to Tag a version?

Reply:
Thanks :)

Reply:
The developer made a v0.1.0 release, which should resolve this problem! :)